### PR TITLE
6 Axis (facebuttons + bumpers) + gripper (RB+LB)

### PR DESCRIPTION
**Pull request by Wojtuz**
As in title

### CODE REVIEW

**Comment by Wojtuz** [comment]
Tested sending msgs to broker.hivemq.com

File: MVVM/View/MainView.tscn [hunk 1]
```diff
@@ -46,15 +46,14 @@ keycode = 83
...
-[node name="MainView" type="Control" node_paths=PackedStringArray("imTextureRect", "RoverModeUIDis", "GrzybUIDis", "MissionStatusUIDis", "ShowSettingsBtn", "ShowVelMonitor", "ShowMissionControlBrn", "SettingsManagerNode", "MissionControlNode", "FancyDebugViewRLab", "VelMonitor", "ZedMonitor")]
+[node name="MainView" type="Control" node_paths=PackedStringArray("RoverModeUIDis", "GrzybUIDis", "MissionStatusUIDis", "ShowSettingsBtn", "ShowVelMonitor", "ShowMissionControlBrn", "SettingsManagerNode", "MissionControlNode", "FancyDebugViewRLab", "VelMonitor", "ZedMonitor")]
 layout_mode = 3
 anchors_preset = 15
 anchor_right = 1.0
 anchor_bottom = 1.0
 grow_horizontal = 2
 grow_vertical = 2
 script = ExtResource("1_84vv0")
-imTextureRect = NodePath("CameraView")
```
Comment (wisniax) [inline MVVM/View/MainView.tscn:57]: Are u sure?

File: project.godot [hunk 2]
```diff
@@ -177,11 +178,15 @@ manipulator_axis_5={
 "events": [Object(InputEventJoypadButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"button_index":9,"pressure":0.0,"pressed":true,"script":null)
 ]
 }
-manipulator_gripper={
+manipulator_axis_6={
 "deadzone": 0.5,
 "events": [Object(InputEventJoypadButton,"resource_local_to_scene":false,"resource_name":"","device":-1,"button_index":10,"pressure":0.0,"pressed":true,"script":null)
 ]
 }
+manipulator_gripper={
```
Comment (wisniax) [inline project.godot:186]: What's the purpose?

Comment (Wojtuz) [reply project.godot]: Adressed in [c7e1660](https://github.com/wisniax/RoverControlApp/pull/32/commits/c7e16602d1c93990e55a49808a1da0aec6af20b4)

File: MVVM/View/MainView.tscn (hunk 1 above)
Comment (Wojtuz) [reply MVVM/View/MainView.tscn]: Reverted [c7e1660](https://github.com/wisniax/RoverControlApp/pull/32/commits/c7e16602d1c93990e55a49808a1da0aec6af20b4)

File: MVVM/View/MainView.tscn [hunk 3]
```diff
@@ -272,9 +272,9 @@ offset_bottom = -40.0
 [connection signal="toggled" from="HBoxContainer/ShowMissionControl" to="MissionControl" method="set_visible"]
 [connection signal="toggled" from="HBoxContainer/ShowVelMonitor" to="VelMonitor" method="set_visible"]
 [connection signal="toggled" from="HBoxContainer/ShowZedMonitor" to="ZedMonitor" method="set_visible"]
+[connection signal="toggled" from="HBoxContainer/ShowSettingsBtn" to="SettingsManager" method="set_visible"]
```
Comment (Wojtuz) [inline MVVM/View/MainView.tscn:275]: Idk why it jumps like that I dont even touch that button

**Comment by Wojtuz** [comment]
Face buttons and bumpers for individual axis control, RB + LB for gripper control.
Tested working (looks good to me as well). Ready to merge?

**Review by wisniax — APPROVED** [review]
Looks good

File: Core/RoverControllerPresets/ManipulatorControllers/SingleAxisManipulatorController.cs [hunk 4]
```diff
@@ -12,16 +12,24 @@ public ManipulatorControl CalculateMoveVector()
 		if (Mathf.Abs(velocity) < LocalSettings.Singleton.Joystick.Deadzone) 
 			velocity = 0f;
 
-		ManipulatorControl manipulatorControl = new()
+		ManipulatorControl manipulatorControl = null;
```
Comment (AlvaroBajceps) [inline Core/RoverControllerPresets/ManipulatorControllers/SingleAxisManipulatorController.cs:15]: null is redundant needed here
```suggestion
		ManipulatorControl manipulatorControl;
```

**Review by AlvaroBajceps — APPROVED** [review]

**Pull request merged**